source/PlayerInfo.cpp

```diff
@@ -3079,6 +3079,18 @@ void PlayerInfo::Save(const string &path) const
 	out.Write();
 	out.WriteComment("How you began:");
 	startData.Save(out);
+
+	// Write plugins to player's save file for debugging.
+	if(!GameData::PluginAboutText().empty()) {
```

Comment:
Braces go on separate lines.

Reply:
Whoops, tried to remind myself to fix that before pushing, and completely forgot.